scripts/store-stats/store-stats.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Vatstore Metrics Tool
+ *
+ * This tool is designed to periodically gather statistics from a SwingStore SQLite database,
+ * specifically counting rows related to Vat storage. It then exposes these metrics in a format
+ * suitable for scraping by a Prometheus instance.
+ *
+ * Usage:
+ *   node <script-name> <path-to-sqlite-db> [interval-in-milliseconds]
+ *
+ * <script-name> - Name of this script.
+ * <path-to-sqlite-db> - The path to the SQLite database to be monitored.
+ * [interval-in-milliseconds] - Optional. How often to fetch data from the database. Defaults to 10 minutes.
+ */
+
+import { MeterProvider } from '@opentelemetry/sdk-metrics';
+import { PrometheusExporter } from '@opentelemetry/exporter-prometheus';
+import sqlite3 from 'sqlite3';
+
+const DB_READ_INTERVAL = 10 * 60 * 1000; // 10 minutes
+const PORT = 9184;
+const ENDPOINT = '/metrics';
+const MAX_VATS = 80;
+
+const dbFilePath = process.argv[2];
+if (!dbFilePath) {
+  console.error('SwingStore SQLite database file is required argument.');
+  process.exit(1);
+}
+
+const dbReadInterval = parseInt(process.argv[3], 10) || DB_READ_INTERVAL;
+
+const vatstores = [];
+
+const db = new sqlite3.Database(dbFilePath);
+const getVatstoreStats = async () => {
+  for (let id = 1; id <= MAX_VATS; id++) {
+    const query = `SELECT COUNT(*) as keyCount FROM kvStore WHERE key LIKE 'v${id}.vs.%'`;
```

Comment:
This is of no consequence here, since `id` is not controllable, but as a matter of habit we should avoid string interpolation and use [parameterized queries](https://github.com/TryGhost/node-sqlite3/wiki/API#runsql--param---callback) or [prepared statements](https://github.com/TryGhost/node-sqlite3/wiki/API#preparesql--param---callback) instead to avoid SQL injection.